Sources/StreamChatSwiftUI/ViewFactory.swift

```diff
@@ -637,7 +637,8 @@ public protocol ViewFactory: AnyObject {
         placeholder: String,
         editable: Bool,
         maxMessageLength: Int?,
-        currentHeight: CGFloat
+        currentHeight: CGFloat,
+        onImagePasted: ((UIImage) -> Void)?
```

Comment:
@anh-ngquang Thanks for the PR 👍 The implementation will probably require some changes because this is a breaking change. We will take it from here once we have time. We are planning on working on this in the next sprint.